packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts

```diff
@@ -136,52 +137,45 @@ export class ComponentUtils {
     const describeSObjectFields = await connection.describe(sObjectName);
     const describeSObjectFieldsList = describeSObjectFields.fields;
     const result = {status: 0, result: describeSObjectFieldsList};
-    const jsonResult = JSON.stringify(result);
+    const jsonResult = JSON.stringify(result, null, 2);
     fs.writeFileSync(componentsPath, jsonResult);
     return jsonResult;
   }
 
   public async loadComponents(
     defaultOrg: string,
     metadataType: string,
-    sObject?: string,
+    folderName?: string,
     forceRefresh?: boolean
   ): Promise<string[]> {
     const componentsPath = await this.getComponentsPath(
       metadataType,
       defaultOrg,
-      sObject
+      folderName
     );
 
     let componentsList: string[];
     const connection = await workspaceContext.getConnection();
 
-    if (metadataType === 'CustomObject' && sObject) {
+    if (metadataType === 'CustomObject' && folderName) {
       if (forceRefresh || !fs.existsSync(componentsPath)) {
-        componentsList = await this.fetchCustomObjectsFields(sObject, connection, componentsPath);
+        componentsList = await this.fetchCustomObjectsFields(folderName, connection, componentsPath);
```

Comment:
Since `fetchMetadataComponents()` has `folderName` passed as the last arg (after connection and componentsPath), how about changing `fetchCustomObjectsFields()` and `fetchExistingCustomObjectsFields()` so it is passed after the others?  There would be more symmetry between the functions then.